gamemodes/terrortown/gamemode/shared/sh_sprint.lua

```diff
@@ -46,6 +46,9 @@ if SERVER then
 		ply.isSprinting = bool
 	end)
 else
+
+	local disable_doubletap_sprint = CreateClientConVar("ttt2_disable_doubletap_sprint", "0", true, true)
```

Comment:
Userinfo (the fourth parameter) should be false. I would personally also add a helptext and min 0 max 1.